Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/TotScatCalculateSelfScattering.py

```diff
@@ -56,12 +56,15 @@ def PyInit(self):
             defaultValue="",
             doc="Sample Temperature in Kelvin. Required for 2nd order Placzek correction if not " "using Sample Logs.",
         )
+        self.declareProperty(name="ApplyPerDetector", defaultValue=False, doc="Apply the correction to unfocussed data.")
```

Comment:
It seems there is a unit test for this file, but it's a bit light. I think given the nature of this algorithm a systems test covers this.